### PR TITLE
Add a page with past members

### PR DESCRIPTION
**Pull request by matteomancini**
I've added a "Past members" page that should be painless to be extended in the next years, for each past year it is a matter of adding a markdown file in the `content/pastmembers` folder and the actual data for the past members in the `data/team` folder. Details about the current members are stored as before in the `config.yaml` file.
I've also updated the hackathon and osr links so now they directly go the respective websites.

### CODE REVIEW

**Review by 62442katieb — APPROVED** [review]
lgtm!

File: themes/hugo-fresh/assets/fresh/partials/_sections.scss [hunk 1]
```diff
@@ -1,85 +1,94 @@
...
+}
+
+.year-link {
+        color: $muted-grey;
+        &.is-secondary {
+            &:hover {
+                color: $secondary !important;
+            }
+        }
 }
```
Comment (sofievalk) [inline themes/hugo-fresh/assets/fresh/partials/_sections.scss]: wants a new line (also a warning later -- but guess it is not that  important)

**Review by sofievalk — APPROVED** [review]
Do we also have pictures of all old teams? Or just of current?

**Comment by matteomancini** [comment]
Yes, everyone is there! :)

**Pull request merged**